sdk/identity/Azure.Identity/tests/ProcessRunnerTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.Identity.Tests
+{
+    [TestFixture(true)]
+    [TestFixture(false)]
+    public class ProcessRunnerTests
```

Comment:
I wonder if we can test with a real processes here.

Reply:
Something like `cmd.exe`?

Reply:
Yeah, `cmd /c "echo blalb"` for windows and `sh "echo lalala"`

Reply:
Done